### ISSUE TITLE
Errors with JAWS and IE11

### ISSUE THREAD

**Issue by zachberry** [ucfopen/Obojobo#683]
Keeping a catalog of issues found with using JAWS (tested in IE11):

* [ ] - The `name` attributes of inputs (such as radio buttons in multiple choice questions) are read. As these are usually UUIDs it is very disorienting.
* [ ] - Math content not read when in a list (See first page of the test-object)
* [ ] - Math content in a question is read even if that question has not been "flipped" yet
* [ ] - Flipping a question focuses on the first element in the question but does not read the element
* [ ] - Feedback and answer choices in questions seem to be read multiple times

This is not an exhaustive list, just some quick preliminary testing. Additional JAWS issues can be added to this issue.
